solidity/contracts/adapters/UniswapV3Adapter.sol

```diff
@@ -137,6 +138,11 @@ contract UniswapV3Adapter is AccessControl, IUniswapV3Adapter {
     emit DenylistChanged(_pairs, _denylisted);
   }
 
+  /// @inheritdoc IERC165
+  function supportsInterface(bytes4 interfaceId) public view virtual override(AccessControl, BaseOracle) returns (bool) {
+    return AccessControl.supportsInterface(interfaceId) || BaseOracle.supportsInterface(interfaceId);
+  }
```

Comment:
The CLAdapter is the same as the ITokenPriceOracle, except for one function. I don't think it makes sense to create the interface and add support for it (specially since we know it's going to die soon)

But I did add this for the Uni V3 adapter